src/Simplex/Chat/Controller.hs

```diff
@@ -240,6 +248,12 @@ data ChatResponse
   | CRPendingSubSummary {pendingSubStatus :: [PendingSubStatus]}
   | CRSndFileSubError {sndFileTransfer :: SndFileTransfer, chatError :: ChatError}
   | CRRcvFileSubError {rcvFileTransfer :: RcvFileTransfer, chatError :: ChatError}
+  | CRCallInvitationSent {contact :: Contact}
```

Comment:
Is there a reason to use the specific response for invitation and not CRCmdOk like for other commands?

Reply:
callId was used internally in the beginning, can be replaced with Ok too